### PR TITLE
Update various Trinkey pins and add _pixelbuf

### PR DESCRIPTION
**Pull request by dhalbert**
Rotaryio Trinkey pins were from an older rev board.

Tested with a mounted encoder. `.position` increases clockwise with `IncrementalEncoder(board.ROTA, board.ROTB)`, as expected. Also tested `board.SWITCH` and `board.TOUCH`.

### CODE REVIEW

**Review by ladyada — DISMISSED** [review]

**Comment by dhalbert** [comment]
This is ready for re-review. The single build failure was a GitHub Actions problem.

**Review by ladyada — APPROVED** [review]

File: ports/atmel-samd/boards/neopixel_trinkey_m0/mpconfigboard.mk [hunk 1]
```diff
@@ -20,8 +20,6 @@ CIRCUITPY_AUDIOCORE = 0
 CIRCUITPY_BUSIO = 0
 
 CIRCUITPY_PIXELBUF = 1
-CIRCUITPY_USB_MIDI = 1
-CIRCUITPY_TOUCHIO = 1
```
[the remark this answers is not in the document]
Comment (dhalbert) [reply ports/atmel-samd/boards/neopixel_trinkey_m0/mpconfigboard.mk:24]: It is enabled by defaul heret, so it doesn't need to be listed explicitly. Same for MIDI:
https://github.com/adafruit/circuitpython/blob/25a44cb77ff53ade15fdfac09e74d59b4b4b375e/py/circuitpy_mpconfig.mk#L334
And it is not turned off for small SAMD21 builds:
https://github.com/adafruit/circuitpython/blob/25a44cb77ff53ade15fdfac09e74d59b4b4b375e/ports/atmel-samd/mpconfigport.mk#L31-L61

[the remark this answers is not in the document]
Comment (ladyada) [reply ports/atmel-samd/boards/neopixel_trinkey_m0/mpconfigboard.mk:24]: ah ok! thanks :)

**Comment by dhalbert** [comment]
Current modules list from https://circuitpython.readthedocs.io/en/latest/shared-bindings/support_matrix.html (built after this PR merged)

image
image

**Pull request merged**